go/BUILD.bazel

```diff
@@ -0,0 +1,48 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("//go:rules.bzl", "swig_native_go_library")
+load("@bazel_gazelle//:def.bzl", "gazelle")
+
+# for updating dependencies
+gazelle(
+    name = "gazelle-update-repos",
+    args = [
+        "-from_file=go/go.mod",
+        "-to_macro=go/deps.bzl%go_dependencies",
+        "-prune",
+        "-build_file_proto_mode=disable_global",
+    ],
+    command = "update-repos",
+)
+
```

Comment:
Sometimes it's two newlines between rules, sometimes it's one. Let's get more consistency, it's a production code =)